client/src/main/java/com/battleships/start_window/data_insertion/InsertTextData.java

```diff
@@ -0,0 +1,45 @@
+package com.battleships.start_window.data_insertion;
+
+import com.battleships.Translator;
+import com.battleships.start_window.connection.ConnectInfo;
+import com.battleships.start_window.connection.Connection;
+import javafx.beans.binding.Bindings;
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.TextField;
+
+public class InsertTextData {
+    @FXML
+    private Button connectToServerButton;
+    @FXML
+    private Button disconnectButton;
+    @FXML
+    private TextField nameTextField;
+    @FXML
+    private TextField ipTextField;
+
+    @FXML
+    public void initialize() {
+        bindTextFieldsWithTranslation();
+    }
+
+    private void bindTextFieldsWithTranslation() {
```

Comment:
method is doing different stuff than it's name suggests

Reply:
fixed